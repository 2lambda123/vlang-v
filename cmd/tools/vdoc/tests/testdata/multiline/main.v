/*
Hello!
*/
pub fn funky() {
	println('hi')
}

// normal comment 1
/*
multiline comment 2
*/
// normal comment 3
pub fn veryfunky() {
	println('hey!')
}
