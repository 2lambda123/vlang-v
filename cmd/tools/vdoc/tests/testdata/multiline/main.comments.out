module main

fn funky()
    Hello!
fn veryfunky()
    normal comment 1
    
    multiline comment 2
    
    normal comment 3